transports/jms/src/main/java/org/mule/compatibility/transport/jms/JmsMessageDispatcher.java

```diff
@@ -164,25 +164,17 @@ private InternalMessage dispatchMessage(Event event, boolean doSend,
 
       if (useReplyToDestination && replyTo != null) {
         final int timeout = endpoint.getResponseTimeout();
+        try {
```

Comment:
should this class extend the new abstract class to continue testing the non blocking scenarios?